dataprofiler/tests/labelers/test_regex_model.py

```diff
@@ -183,14 +182,17 @@ def test_predict(self, mock_stdout):
                                [0, 1, 0],
                                [0, 1, 0]])
          ]}
-        model_output = model.predict(['   ', 'hello'])
+        with self.assertLogs('DataProfiler.labelers.regex_model',
+                             level='INFO') as cm:
+            model_output = model.predict(['   ', 'hello'])
         self.assertIn('pred', model_output)
+        log_output = ''.join(cm.output)
```

Comment:
if we aren't using this here, i would suggest moving it to just above the assert it is being used with for clarity.

Reply:
Moved